db/cache.py

```diff
@@ -21,7 +21,7 @@
 _glob_namespace = "AB"
 
 
-def init(servers, namespace="AB", debug=0):
+def init(servers, namespace="MESSY", debug=0):
```

Comment:
in other places we're using MyB (or MYB) as the standard letters + B musicbrainz format